addons/event/views/event_ticket_views.xml

```diff
@@ -77,8 +77,8 @@
                 <field name="start_sale_datetime" optional="show"/>
```

Comment:
Same thing here, you can squash all 3 commits.

As a general remark, you don't have to be that specific about your changes in your commit message.
It can be something like:

> This commit applies some general improvements to the event form, notably:
> - Swap some fields to better positions
> - Reword a few field labels / stats buttons
> - ...

(Did not write a complete one but you get the idea).

The actual changes can be found by reading the code.
Especially when they're very straightforward, such as labels changes.